com.trekglobal.idempiere.rest.api/src/com/trekglobal/idempiere/rest/api/json/RestUtils.java

```diff
@@ -170,12 +170,21 @@ public static Query getQuery(String tableName, String whereClause, List<Object>
 			whereClause = whereClause + "AD_Language=?";
 			params.add(Env.getAD_Language(Env.getCtx()));
 		}
+		
+		if(includeInactive!=null && includeInactive.equals("only")) {
+			whereClause += "isActive=?";
+			params.add("N");
+		}
 
 		Query query = new Query(Env.getCtx(), table, whereClause, null);
 
-		query.setApplyAccessFilter(true, false)
-			.setOnlyActiveRecords(true)
-			.setParameters(params);
+		query.setApplyAccessFilter(true, false);
+		
+		if(includeInactive==null)
+			query.setOnlyActiveRecords(true);
+		
```

Comment:
please clear double empty lines, and also extra spaces or tabs at the end of the added lines.